test/metabase/lib/js_test.cljs

```diff
@@ -668,3 +668,15 @@
               (is (=? (lib/append-stage agg-only)
                       (.-query obj)))
               (is (=? -1 (.-stageIndex obj))))))))))
+
+(deftest ^:parallel display-info->js-test
+  (testing "all data structures are converted correctly"
+    (is (goog.object/equals
```

Comment:
This wants to use the `js=` function elsewhere in this file. Like `(is (js= #js {:things :here} (lib.js/display-info->js ...)))`.